malcolm/modules/ADPandABlocks/doublebuffer.py

```diff
@@ -238,7 +239,19 @@ def configure(self, rows_generator: Iterator[SequencerRows]) -> None:
             # The template sets repeats to 1 however older designs may not have
             # been updated.
             for table in self._table_map.values():
-                table.repeats.put_value(1)
+
+                if table.repeats.value != 1:
+                    logging.warning(
+                        "Repeats on Seq table {} should be set to '1', Malcolm has"
+                        "overwritten value for this scan".format(table.label.value)
+                    )
+                    table.repeats.put_value(1)
+                if table.prescale.value != 0:
```

Comment:
0 or 1 are fine

Reply:
If it is set to 1 it will still scale the timings by the Prescale Units (s, ms, or us). I think the values being inputted to the table are in Clock Ticks, so even when prescale is 1 us the timings do not work.

Reply:
Good point, I'd forgotten about units, 0 or 1 clock ticks are fine, but the units complicates checking. I agree that it makes sense to require a value of 0.